.github/workflows/kubeflow.yml

```diff
@@ -0,0 +1,58 @@
+name: Kubeflow
+on: workflow_dispatch
+
+jobs:
+  integration_tests:
+    runs-on: ubuntu-latest
+    env:
+      ZENML_DEBUG: 1
+      ZENML_ANALYTICS_OPT_IN: false
+
+    steps:
+      # Workaround from FuseML (https://github.com/fuseml/fuseml/blob/main/.github/workflows/ci.yml)
+      # as the TF images are too large for the GH action runner disk
+      - name: Free disk space
+        run: |
+          echo "Available storage:"
+          df -h
+          echo
+          echo -n "  Removing: .NET (frees ~22GB)"
+          sudo rm -rf /usr/share/dotnet
+          echo "... done"
+          echo -n "  Removing: Android"
+          sudo rm -rf /usr/local/lib/android
+          echo "... done"
+          echo
+          echo "Available storage:"
+          df -h
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install K3D and kubectl
+        run: |
+          curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+      - name: Install poetry
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install "poetry"
+
+      - name: Install Dependencies
+        run: |
+          python -m poetry install
+          python -m poetry run zenml integration install kubeflow -f
+
+      - name: Build docker image
+        # this step can be much faster if we use an image that copies the
```

Comment:
Very important. Maybe you should even make this a TODO item.

Reply:
Unfortunately our todo script is too stupid to detect non-python files atm :P
I'll create a jira issue directly to make sure we implement this soon